### PR TITLE
Adding gts 0.7.6.

### PR DESCRIPTION
**Pull request by seanyen**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by seanyen** [comment]
@conda-forge/staged-recipes This is ready for review and merge. Thanks!

File: recipes/gts/config.in.win32.patch [hunk 1]
```diff
@@ -0,0 +1,13 @@
+--- src/config.h.win32
++++ src/config.h.win32
+@@ -7,8 +7,8 @@
+ #define STDC_HEADERS 1
+ 
+ #define GTS_MAJOR_VERSION 0
+-#define GTS_MINOR_VERSION 6
+-#define GTS_MICRO_VERSION 0
++#define GTS_MINOR_VERSION 7
++#define GTS_MICRO_VERSION 6
```
Comment (scopatz) [inline recipes/gts/config.in.win32.patch]: Is the version number really wrong in the source code?

File: recipes/gts/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}  # [not win]
+    - {{ compiler('c') }}    # [not win]
+    - vs2017_win-64          # [win]
+    - cmake       # [win]
+    - ninja       # [win]
+    - make        # [not win]
+    - pkgconfig   # [not win]
```
Comment (scopatz) [inline recipes/gts/meta.yaml]: I think you want `pkg-config`

File: recipes/gts/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+test:
+  commands:
+    - test -f ${PREFIX}/lib/libgts.so  # [linux]
+    - test -f ${PREFIX}/lib/libgts-0.7.5.0.1.dylib  # [osx]
+    - if not exist %LIBRARY_PREFIX%\\bin\\gts.dll exit 1  # [win]
+
+about:
+  home: http://gts.sourceforge.net/
+  license: LGPLv2
+  license_file: COPYING
```
Comment (scopatz) [inline recipes/gts/meta.yaml]: You need to have a `license_family` entry

File: recipes/gts/config.in.win32.patch (hunk 1 above)
Comment (seanyen) [reply recipes/gts/config.in.win32.patch]: Yeah, it does have a `config.in` as a template but it is for non-Windows, but the Windows one `config.in.win32` appears to be a hand-crafted one and out-of-dated on the version...

Comment (scopatz) [reply recipes/gts/config.in.win32.patch]: Weird.... you should file a bug

File: recipes/gts/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}  # [not win]
+    - {{ compiler('c') }}    # [not win]
+    - vs2017_win-64          # [win]
+    - cmake       # [win]
+    - ninja       # [win]
+    - make        # [not win]
+    - pkgconfig
```
Comment (scopatz) [inline recipes/gts/meta.yaml]: Should be `pkg-config`

**Comment by isuruf** [comment]
See https://github.com/conda-forge/autotools_clang_conda-feedstock/blob/master/recipe/meta.yaml#L34-L60 if you want to use autotools with windows

File: recipes/gts/meta.yaml (hunk 4 above)
Comment (seanyen) [reply recipes/gts/meta.yaml]: ah, I misread the last comment. I fixed the typo.

File: recipes/gts/config.in.win32.patch (hunk 1 above)
Comment (seanyen) [reply recipes/gts/config.in.win32.patch]: Filed one...https://sourceforge.net/p/gts/bugs/73/

**Comment by seanyen** [comment]
> See https://github.com/conda-forge/autotools_clang_conda-feedstock/blob/master/recipe/meta.yaml#L34-L60 if you want to use autotools with windows

Thanks for the pointer. That looks interesting. I will take a look on that, but I don't have much cycles now to rework this pull request, so probably I will do it later.

File: recipes/gts/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+    - pkg-config  # [not win]
+  host:
+    - glib
+  run:
+    - glib
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/libgts.so  # [linux]
+    - test -f ${PREFIX}/lib/libgts-0.7.5.0.1.dylib  # [osx]
```
Comment (isuruf) [inline recipes/gts/meta.yaml]: Is the version in the name intended here?

Comment (seanyen) [reply recipes/gts/meta.yaml]: No, let me see if the build still works without  the version suffix check.

Comment (seanyen) [reply recipes/gts/meta.yaml]: @isuruf Looks I don't need the version check. :)

**Pull request merged**